src/domain/flow-system/domain/src/services/flow/flow_service.rs

```diff
@@ -53,10 +53,10 @@ pub trait FlowService: Sync + Send {
     /// Returns initiators of flows associated with a given dataset
     /// ordered by creation time from newest to oldest.
     /// Applies specified filters/pagination
-    async fn list_all_flow_initiators_by_dataset(
+    async fn get_all_flow_initiators_by_dataset(
         &self,
         dataset_id: &DatasetID,
-    ) -> Result<FlowInitiatorListing, ListFlowsByDatasetError>;
+    ) -> Result<Vec<AccountID>, ListFlowsByDatasetError>;
```

Comment:
The method signature was changed with the need to get away from `'a`.

I haven't found a way to do borrow from a temporarily existing transactional catalog. Previously the borrowing was taken from the `FlowServiceImpl::flow_event_store` field, but now the store is always retrieved dynamically.

TODO 
- [ ] Add a link to usage

Reply:
CC @rmn-boiko

Reply:
Usage: https://github.com/kamu-data/kamu-cli/pull/650/files#r1619221031